doc/src/manual/methods.md

```diff
@@ -241,23 +240,18 @@ julia> g(2, 3.0)
 8.0
 
 julia> g(2.0, 3.0)
-ERROR: MethodError: g(::Float64, ::Float64) is ambiguous. Candidates:
-  g(x, y::Float64) in Main at none:1
-  g(x::Float64, y) in Main at none:1
```

Comment:
They are run in Main, but all functions are defined in its own module. I think... These two tests both pass at least:

````
```jldoctest
julia> current_module()
Main
```

```jldoctest currentmod
julia> current_module()
Main
```
````

Note that this is the only one where this have been a problem. Perhaps we can skip the stacktrace for this one? The text section right after explains what is going on as well.